### PR TITLE
Write Tests for SuggestCommentScore

### PR DESCRIPTION
**Pull request by chrismohrii**
Writes some tests for SuggestCommentServlet.java to to check that it works correctly. I tried to mimic the style of the AnalyzeComment Perspective tests as much as possible. These are not in this branch because merging has been slow, and there will be a couple identical files to delete when everything is eventually merged. 
Note: Holding off on merging until the parent branch is approved

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java [hunk 1]
```diff
@@ -89,7 +102,7 @@ private String makePerspectiveJson(String text, String lang, HashMap<String, Str
       JSONObject value = new JSONObject();
       value.put("value", feedback.get(suggestion));
       JSONObject summaryScoreValue = new JSONObject();
-      summaryScoreValue.put("summaryScore", value);
+    //  summaryScoreValue.put("summaryScore", value);
```
Comment (danieldortega77) [inline capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: Is this supposed to be commented out?

**Review by danieldortega77 — APPROVED** [review]

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java (hunk 1 above)
Comment (chrismohrii) [reply capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: No, thanks for catching this!

**Review by olu144 — APPROVED** [review]

**Review by dborkan — APPROVED** [review]
LGTM

File: capstone/src/test/java/com/google/sps/servlets/SuggestCommentServletTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  @Before
+  public void setUp() {
+    this.stringWriter = new StringWriter();
+    this.writer = new PrintWriter(stringWriter);
+    this.request = Mockito.mock(HttpServletRequest.class);
+    this.response = Mockito.mock(HttpServletResponse.class);
+  }
+
+  @Test
+  public void SuggestCommentCall() throws IOException{
```
Comment (dslucas) [inline capstone/src/test/java/com/google/sps/servlets/SuggestCommentServletTest.java]: nit: Let's use camel case for the test method names.

**Review by dslucas — APPROVED** [review]

**Pull request merged**